src/unicode/unicode.cr

```diff
@@ -1,13 +1,6 @@
-# Provides methods that answer questions about unicode characters,
-# and the `Unicode::CaseOptions` enum for special case conversions
-# like Turkic.
```

Comment:
I'd just leave this part and remove the rest.

Reply:
This is included in the public comment now.

Reply:
Got misled by the diff, right.